OrbitGl/TimeGraph.cpp

```diff
@@ -136,19 +136,22 @@ void TimeGraph::ZoomAll() {
   }
 }
 
-//-----------------------------------------------------------------------------
-void TimeGraph::Zoom(const TextBox* a_TextBox) {
-  const Timer& timer = a_TextBox->GetTimer();
-
-  double start = MicroSecondsFromTicks(m_SessionMinCounter, timer.m_Start);
-  double end = MicroSecondsFromTicks(m_SessionMinCounter, timer.m_End);
+void TimeGraph::Zoom(TickType min, TickType max) {
+  double start = MicroSecondsFromTicks(m_SessionMinCounter, min);
+  double end = MicroSecondsFromTicks(m_SessionMinCounter, max);
```

Comment:
I really liked the separation you made with Zoom.

I would like to have a parameter here (with a default value) that defines the percentage of the screen that the text boxes will occupy. It is actually 0.9 (or more precisely 1 / 1.1).

In addition, but less important, we can give the user the possibility to change this parameter (make it smaller or larger), so that he can see more to the sides after each zoom (if they wish).

Reply:
Yes, definitely! We will need some UI elements for that, so I think it would be good to postpone to a later PR.